numpy/random/mtrand/mtrand.pyx

```diff
@@ -1525,6 +1525,13 @@ cdef class RandomState:
 
         anywhere within the interval ``[a, b)``, and zero elsewhere.
 
+        Although the method signature suggests that ``low`` should be
+        less than ``high``, this function is well-defined for the
+        cases when ``low`` == ``high`` and ``low`` > ``high``. In
+        the former case, an ndarray containing of ints all equal to
+        ``low`` will be returned. In the latter case, numbers will
+        be drawn from the uniform distribution [``high``, ``low``).
```

Comment:
I think `high` is always the open side of the interval, and `low` always the closed one, so if `low > high`, then the interval is `(high, low]`.